src/main/java/seedu/address/model/person/Id.java

```diff
@@ -5,11 +5,11 @@
 
 /**
  * Represents a Person's address in the address book.
- * Guarantees: immutable; is valid as declared in {@link #isValidAddress(String)}
+ * Guarantees: immutable; is valid as declared in {@link #isValidId(String)}
  */
-public class Address {
+public class Id {
 
-    public static final String MESSAGE_CONSTRAINTS = "Addresses can take any values, and it should not be blank";
+    public static final String MESSAGE_CONSTRAINTS = "Ids must be of format AXXXXXXXX, and should not be blank";
```

Comment:
The id constraint might need to be flexible in the future to cater to nus staff.